pkg/component/etcd/bootstrap.go

```diff
@@ -290,7 +318,7 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 									},
 								},
 								Ports: []corev1.ContainerPort{{
-									ContainerPort: 9569,
```

Comment:
What was this port to begin with? Is it no longer needed so that it can just be replaced?
I do not see this port being open in a productive `etcd-druid` installation. Hence, I am a bit confused...

Reply:
I also don't know what this port was about. We are not using this port anywhere in Druid. So, after discussing with other team members, I removed it

Reply:
Thanks for the honesty.